src/core/reducers/index.ts

```diff
@@ -1,6 +1,7 @@
 import * as redux from 'redux';
-import undoable from 'redux-undo';
+import { default as undoable, includeAction } from 'redux-undo';
```

Comment:
I think the style we use in other places is `import undoable, { includeAction } from 'redux-undo';`

Reply:
I need to import 'redux-undo'.default as undoable, therefore the current style. Maybe there's a better way of writing it?

Reply:
That's what my syntax is doing